test/blockchain/log-processors/market-created.js

```diff
@@ -123,10 +128,11 @@ describe("blockchain/log-processors/market-created", () => {
             tag2: "TEST_TAG_2",
             volume: 0,
             sharesOutstanding: 0,
+            reportingState: "PRE_REPORTING",
```

Comment:
Can we have these reporting states recorded as a typescript Enum so that we get some value checking in the compiler?